ui/js/components/perf/compare.js

```diff
@@ -64,7 +64,7 @@ treeherder.component('phCompareTable', {
             }
             return results.filter((result) => {
                 const testCondition = `${key} ${($attrs.type === 'trend') ? result.trendResult.name : result.name}`;
-                return _.every(ctrl.filterOptions.filter.split(' '), function (matchText) {
+                return ctrl.filterOptions.filter.split(' ').every(function (matchText) {
```

Comment:
Would you mind also switching usage of ``function`` to ``() =>`` syntax in these 3 cases?

Reply:
No problem, can I omit the parentheses when there is only one parameter?
Example: `x => {...}` instead of `(x) => {...}`

Reply:
Never mind, I'm keeping parentheses 👍